### PR TITLE
[v2.5.6] Revert new changes for Logging 0.2.3 and add them to Logging 0.2.4

### PR DESCRIPTION
**Pull request by nickgerace**
Additional commits were added to Logging 0.2.3 for Rancher v2.4.13. However, Logging 0.2.3 released with Rancher v2.5.5. Since the changes are [in the v2.4 system-charts branch](https://github.com/rancher/system-charts/pull/396), we should "revert" the 0.2.3 changes and create a 0.2.4 chart for Logging.

Original commits ([src](https://github.com/rancher/system-charts/commits/dev-v2.5/charts/rancher-logging/0.2.3)):
- `d149753c48a48b99a32c876dc6cdf8a154183d26`
- `a58b7a55c3c7dbcc6e18ae4cedca8ad520707842`
- `db1f6ced7c82bd55768287eda520686c442ac3a2`

This PR is split into **three** commits...
1. "Revert" changes in logging 0.2.3
2. Copy logging 0.2.3
3. Add logging 0.2.4 (with "reverted" changes)

Relevant issue: https://github.com/rancher/rancher/issues/30888

### CODE REVIEW

File: charts/rancher-logging/0.2.4/charts/fluentd/charts/fluentd-linux/values.yaml [hunk 1]
```diff
@@ -51,6 +51,6 @@ configmapReload:
   name: reloader
   image:
     repository: rancher/configmap-reload
```
Comment (cbron) [inline charts/rancher-logging/0.2.4/charts/fluentd/charts/fluentd-linux/values.yaml:53]: shouldn't line 6 above swap to 0.24 ?

**Review by cbron — CHANGES_REQUESTED** [review]
This: https://github.com/rancher/system-charts/pull/425/commits/63ab9347c7ee36148634fb08c16cd1ac4afc6f40
Should be inverse of this: https://github.com/rancher/system-charts/pull/425/commits/93d2e8ec282692a3e7c86a49983e1bac7ec5ecaf

Plus added chart changes for bump, which it looks like are just the chart version

File: charts/rancher-logging/0.2.4/charts/fluentd/charts/fluentd-linux/values.yaml (hunk 1 above)
Comment (nickgerace) [reply charts/rancher-logging/0.2.4/charts/fluentd/charts/fluentd-linux/values.yaml:53]: huh. Don't know how this happened. Fixed.

**Comment by nickgerace** [comment]
> This: [63ab934](https://github.com/rancher/system-charts/commit/63ab9347c7ee36148634fb08c16cd1ac4afc6f40)
> Should be inverse of this: [93d2e8e](https://github.com/rancher/system-charts/commit/93d2e8ec282692a3e7c86a49983e1bac7ec5ecaf)
> 
> Plus added chart changes for bump, which it looks like are just the chart version

@cbron I'm believe is this already the case barring the one spot. Is there a place where the changes look misplaced or out of order?

**Review by cbron — APPROVED** [review]
Fluentd-linux version is now v0.1.24, looks good now

**Review by rmweir — APPROVED** [review]
LGTM

**Pull request merged**